src/BeadingStrategy/BeadingStrategy.cpp

```diff
@@ -8,31 +8,25 @@
 namespace cura
 {
 
-BeadingStrategy::BeadingStrategy
-(
-    coord_t optimal_width,
-    Ratio wall_split_middle_threshold,
-    Ratio wall_add_middle_threshold,
-    coord_t default_transition_length,
-    float transitioning_angle
-) :
-    optimal_width(optimal_width),
-    wall_split_middle_threshold(wall_split_middle_threshold),
-    wall_add_middle_threshold(wall_add_middle_threshold),
-    default_transition_length(default_transition_length),
-    transitioning_angle(transitioning_angle)
+BeadingStrategy::BeadingStrategy(coord_t optimal_width, Ratio wall_split_middle_threshold, Ratio wall_add_middle_threshold, coord_t default_transition_length, float transitioning_angle)
```

Comment:
When lines exceed max line length char the arguments should be aligned.
```
BeadingStrategy::BeadingStrategy(
    coord_t optimal_width, 
    Ratio wall_split_middle_threshold, 
    Ratio wall_add_middle_threshold, 
    coord_t default_transition_length, 
    float transitioning_angle
)
```
Arguments should be aligned +4 spaces from the start of the function name.